### ISSUE TITLE
Possible to support compiled

### ISSUE THREAD

**Issue by hc-codersatlas** [styled-system/styled-system#1532]
Hi, [compiled](https://compiledcssinjs.com) is a CSS in JS solution similar to emotion / styled-components except that it compiles to real css under the hood. There's even a [migration guide](https://compiledcssinjs.com/docs/migrating). Following that - would this work with styled-system?

Thanks.

**Comment by verystack** [comment]
Bump on this

**Comment by mliq** [comment]
Has anyone tried it to find out if it would work?

**Comment by verystack** [comment]
Yeah does not work. Compiled does static analysis - unlike almost every other precompiled lib. So lots of bugs if you try to use/roll something like this.
